src/DADynamoConsole/Program.cs

```diff
@@ -0,0 +1,4 @@
+﻿using System.Reflection;
+
+Assembly.LoadFrom(@"C:\Program Files\Autodesk\Revit 2025\RevitAPI.dll");
```

Comment:
are these console applications / entry points used? How does DA call them? Won't these not run the in context of Revit but as separate processes?